File: Transpiled/Core/OffsetDatum.cpp
#include "OffsetDatum.hpp"

namespace NumbatLogic
{
	class OffsetDatum;
}
#line 0 "/home/cliffya/git/Lang/Source/Core/OffsetDatum.nll"
namespace NumbatLogic
{
#line 3 "/home/cliffya/git/Lang/Source/Core/OffsetDatum.nll"
#line 7 "/home/cliffya/git/Lang/Source/Core/OffsetDatum.nll"
	OffsetDatum* OffsetDatum::Create(OffsetDatum* pOther)
	{
		OffsetDatum* pThis = new OffsetDatum();
		if (pOther != 0)
			pThis->Set(pOther);
		NumbatLogic::OffsetDatum* __3749477135 = pThis;
#line 12 "/home/cliffya/git/Lang/Source/Core/OffsetDatum.nll"
		pThis = 0;
#line 12 "/home/cliffya/git/Lang/Source/Core/OffsetDatum.nll"
		return __3749477135;
	}

	void OffsetDatum::Set(OffsetDatum* pOther)
	{
		m_nOffset = pOther->m_nOffset;
	}

#line 3 "/home/cliffya/git/Lang/Source/Core/OffsetDatum.nll"
	OffsetDatum::OffsetDatum()
	{
		m_nOffset = 0;
	}

}

File: Transpiled/Core/OffsetDatum.cs
#line 1 "../../../Source/Core/OffsetDatum.nll"
namespace NumbatLogic
{
	class OffsetDatum
	{
		public int m_nOffset;
#line 7 "../../../Source/Core/OffsetDatum.nll"
		public static OffsetDatum Create(OffsetDatum pOther)
		{
			OffsetDatum pThis = new OffsetDatum();
			if (pOther != null)
				pThis.Set(pOther);
			NumbatLogic.OffsetDatum __3749477135 = pThis;
#line 12 "../../../Source/Core/OffsetDatum.nll"
			pThis = null;
#line 12 "../../../Source/Core/OffsetDatum.nll"
			return __3749477135;
		}

		public void Set(OffsetDatum pOther)
		{
			m_nOffset = pOther.m_nOffset;
		}

#line 3 "../../../Source/Core/OffsetDatum.nll"
		public OffsetDatum()
		{
		}

	}
}

